### PR TITLE
Feature/db migrate

### PR DESCRIPTION
**Pull request by anthraxx**

### CODE REVIEW

File: migrations/alembic.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+qualname =
+
+[logger_sqlalchemy]
+level = WARN
+handlers =
+qualname = sqlalchemy.engine
+
+[logger_alembic]
+level = INFO
+handlers =
```
Comment (SantiagoTorres) [inline migrations/alembic.ini]: Are these needed? I see they are assigning nothing...

File: tracker/cli/db.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    echo('Initializing database...', nl=False)
+    db.create_all()
+    echo('done')
+
+
+@db_cli.command()
+@option('--integrity/--no-integrity', default=True, help='Check database integrity.')
+@option('--foreign-key/--no-foreign-key', default=True, help='Check foreign keys.')
+@with_appcontext
+def check(integrity, foreign_key):
```
Comment (SantiagoTorres) [inline tracker/cli/db.py]: Should we add a check to mention that `--no-integrity` + `--no-foreign-key` turns this into a no-op?

File: tracker/cli/setup.py [hunk 3]
```diff
@@ -29,6 +30,23 @@ def setup():
...
 
+@setup.command()
+@option('--purge', is_flag=True, help='Purge all data and tables.')
+@pass_context
+def database(ctx, purge=False):
+    """Initialize the database tables."""
+
+    # Auto rename old database for compatibility
+    db_old = '{}/app.db'.format(basedir)
+    db_new = '{}/tracker.db'.format(basedir)
```
Comment (SantiagoTorres) [inline tracker/cli/setup.py]: is there a reason we moved from `os.path.join` to `format`?

**Comment by jelly** [comment]
I'm missing the new dependencies in the requirements.txt and README.md

**Comment by jelly** [comment]
Maybe we can add migration steps for a new column or table to the README.

File: migrations/README [hunk 4]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
[the remark this answers is not in the document]
Comment (anthraxx) [reply migrations/README]: fixed

**Pull request merged**